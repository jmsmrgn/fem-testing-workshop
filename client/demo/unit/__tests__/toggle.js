import React from 'react'
import {render} from 'enzyme'
import Toggle from '../toggle'

test('the component renders with defaults', () => {
  const wrapper = render(<Toggle onToggle={() => {}}>I am child</Toggle>)
  expect(wrapper).toMatchSnapshotWithGlamor()
})
