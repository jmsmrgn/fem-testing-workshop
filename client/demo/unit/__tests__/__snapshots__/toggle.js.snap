// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component renders with defaults 1`] = `
.css-vp4bp7,
[data-css-vp4bp7] {
  text-align: center;
  display: inline-block;
  margin-bottom: 0px;
  font-size: 14px;
  font-weight: 400;
  line-height: 1.4;
  padding: 6px 12px;
  cursor: pointer;
  border-radius: 4px;
  color: #fff;
  background-color: #337ab7;
  border-color: #285f8f;
}

<button
  class="css-vp4bp7"
  data-test="button"
>
  I am child
</button>
`;
